components/SMSAuthenticateForm.tsx

```diff
@@ -0,0 +1,20 @@
+type SMSProps = React.PropsWithChildren<{
+  memberID: string,
+  orgID: string,
+}>;
+export const SMSAuthenticateForm = ({ memberID, orgID }: SMSProps) => {
+  return (
+    <form method="POST" action="/api/smsmfa/authenticate" className="row">
```

Comment:
Could we add some copy along the lines of "We sent a one-time code to the # we have on file for this account"?

Reply:
https://github.com/stytchauth/stytch-b2b-nextjs-example/pull/20/commits/9ef1a7368fdd456d0c6c5078da564782be60e7ac